src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="get"
+      data-qa="big"
+      class="search search__big"
+    >
+      <label class="label">
+        <img
+          src="images/Search.svg"
+          alt="search"
+          class="icon icon__big"
+        >
+        <input
+        type="text"
+        name="search"
+        autocomplete="off"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="input input__big"
+        >
```

Comment:
```suggestion
        <input
          type="text"
          name="search"
          autocomplete="off"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
          class="input input__big"
        >
```

Reply:
And please do it for all such cases

Reply:
fixed here but not for all cases, left an additional comment

Reply:
changed